src/tests/accommodationTests/editAccommodationTest.spec.js

```diff
@@ -0,0 +1,113 @@
+import mockStore from '../../utilities/tests/mockStore';
+import findByTestAttribute from './../../utilities/tests/findByTestAttribute';
+import accommodationMocks from '../mocks/accommodationsMocks';
+
+const updatedMock = {
+  name: 'Isimbi Hotel',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+  highlights: 'Lorem Ipsum',
+  amenities: 'Lorem Ipasum',
+  images: 'http://res.cloudinary.com/ddypcld8o/image/upload/v1576174843/beyud6wxuuk9id4dggoh.jpg',
+};
+
+const initialState = {
+  accommodation: {
+    accommodation: null,
+    accommodationError: null,
+    status: '',
+    getAccommodation: [],
+    getAccommodationError: {},
+    singleAccommodation: {},
+    singleAccommodationError: {},
+  },
+};
+
+const locations = {
+  data: {
+    data: [{
+      id: 1,
+      name: 'Blantyre',
+    }, {
+      id: 2,
+      name: 'Lilongwe',
+    }, {
+      id: 3,
+      name: 'Zomba',
+    }],
+  }
+};
+
+const matchProps = {
+  match: {
+    params: 'isimbi-hotel'
+  },
+
+};
+
+const updateError = {
+  data: {
+    message: 'It broke'
+  }
+}
+
+const setUp = (initialState = {}, props) => {
+  const store = mockStore(initialState);
+  const wrapper = shallow(<EditAccommodations store={store} {...props} />).childAt(0).dive();
+  return wrapper;
+};
+
+describe('Test Request Component', () => {
+
+  it('Should show loading page', async () => {
+    const component = setUp(initialState, {})
+    expect(findByTestAttribute(component, 'loading-page').length).toBe(1);
+
+  });
+
+  it('Should render correctly', async () => {
+    const component = setUp(initialState, {})
+    component.setProps({ locations: locations, accommodation: accommodationMocks })
+    component.setState({ accommodation: accommodationMocks, isLoading: false })
+    expect(findByTestAttribute(component, 'form').length).toBe(1);
+  });
+
+  it('Should render correctly', async () => {
+    const component = setUp(initialState, {})
+    component.setProps({ locations: locations, accommodation: accommodationMocks, status: 'success' })
+    component.setState({ accommodation: accommodationMocks, isLoading: false })
+    expect(findByTestAttribute(component, 'redirect').length).toBe(1);
+  });
+
+  it('Should call handleSubmit', async () => {
+    const component = setUp(initialState, {});
+    component.setProps({
+      locations: locations, accommodation: accommodationMocks, GetSingleAccommodation: jest.fn(),
+      getLocations: jest.fn(),
+    })
+    component.setState({ accommodation: accommodationMocks, isLoading: false })
+    component.instance().handleChange({ target: { value: 'this hotel', name: 'name' } });
+    component.instance().updateAccommodation({ target: { ame: 'name', }, preventDefault: jest.fn() });
```

Comment:
Add an expect

Reply:
Fixed this, not sure why the `outdated` tag isn't pasted on it.